hardware/main.py

```diff
@@ -3,13 +3,18 @@
 import json
 
 from dotenv import load_dotenv
+from hardware.Utils.utils import get_logger
+
+logger = get_logger("main_log", "main_log")
+logger.info("Started hardware main.py")
```

Comment:
Should the logger be instantiated after the load_dotenv?

Reply:
I thought of doing that. I added logger for logging automatically running the script on raspberry pi boot. The problem is raspberry pi is unable to identify these packages and still failing.

Reply:
ah ok, would we need to add a variable to the `env` file since the call to `get_logger("main_log", "main_log")` will make a call to `os.environ["main_log"]`?

Reply:
Yes, I had that issue. Thanks for pointing that out.